docs/sphinx/user_guide/profiling_with_caliper.rst

```diff
@@ -0,0 +1,179 @@
+.. ##
+.. ## Copyright (c) 2016-25, Lawrence Livermore National Security, LLC
+.. ## and RAJA project contributors. See the RAJA/LICENSE file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _profiling-with-Caliper-label:
+
+************************
+Profiling with Caliper
+************************
+
+The RAJA abstraction layer has dynamic and static plugin support for the Caliper performance library.
+We refer the reader to the :ref:`feat-plugins-label` section for more on RAJA plugins. Caliper is developed at LLNL and freely available on GitHub,
+see `Caliper GitHub <https://github.com/LLNL/Caliper>`_ . Caliper provides a common interface for various vendor profiling tools and its own built-in performance
+reports.
+
+In this section we demonstrate how to configure RAJA with Caliper, run a simple examples with kernel profiling,
+and a quick snapshot of hotspots of using the Thicket libary. The Thicket library is also developed at LLNL and freely available on GitHub,
+see `Thicket GitHub <https://github.com/LLNL/Thicket>`_ .
+This page is tailored to be a quickstart guide, for more detailed tutorials we refer the reader to the Caliper and Thicket tutorials.
```

Comment:
Would it be useful to add links to those tutorials or note where they can be found (e.g., linked to Caliper and Thicket GitHub project pages)?

Reply:
Those all clickable links that will take the user directly to their page.

Reply:
What I meant was change "we refer the reader to the Caliper and Thicket tutorials" to "we refer the reader to the Caliper and Thicket tutorials, which are accessible on the GitHub pages for those projects"